CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.19.0
+* Vsock API call: PUT /vsocks/{id} changed to PUT /vsock and no longer
+  appear to support multiple vsock devices. Any subsequent calls to this API
+  endpoint will override the previous vsock device configuration.
+* Removed 'Halting' and 'Halted' instance states.
```

Comment:
Can you prefix these changes with "Firecracker v0.19 API: ", just so it's apparent that these are changes from the Firecracker VMM itself rather than Go-SDK specific changes?